app/script/calling/entities/Call.js

```diff
@@ -39,156 +39,156 @@ z.calling.entities.Call = class Call {
    * Construct a new call entity.
    *
    * @class z.calling.entities.Call
-   * @param {z.entity.Conversation} conversation_et - Conversation the call takes place in
-   * @param {z.entity.User} creating_user - Entity of user starting the call
-   * @param {string} session_id - Session ID to identify call
-   * @param {z.calling.CallingRepository} calling_repository - Calling Repository
+   * @param {z.entity.Conversation} conversationEntity - Conversation the call takes place in
+   * @param {z.entity.User} creatingUser - Entity of user starting the call
+   * @param {string} sessionId - Session ID to identify call
+   * @param {z.calling.CallingRepository} callingRepository - Calling Repository
    */
-  constructor(conversation_et, creating_user, session_id, calling_repository) {
-    this.conversation_et = conversation_et;
-    this.creating_user = creating_user;
-    this.session_id = session_id;
-    this.calling_repository = calling_repository;
+  constructor(conversationEntity, creatingUser, sessionId, callingRepository) {
+    this.conversationEntity = conversationEntity;
+    this.creatingUser = creatingUser;
+    this.sessionId = sessionId;
+    this.callingRepository = callingRepository;
 
-    const {id: conversation_id, is_group} = conversation_et;
-    const {media_stream_handler, media_repository, self_state, telemetry, user_repository} = this.calling_repository;
+    const {id: conversationId, isGroup} = conversationEntity;
```

Comment:
Conversation entity has not yet been refactored

Reply:
Can't understand, what should I do here?